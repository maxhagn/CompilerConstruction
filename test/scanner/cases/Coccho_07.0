//khfwiuehfbhsbfhsjdbfjshfjsdfhsf
end4545
sdf //dfsdferwer234234_9874h@&#%^@*$%/*-
// //DOUBLE COMMENT