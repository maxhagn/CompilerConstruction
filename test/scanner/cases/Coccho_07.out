id end4545
id sdf